install/third-party/Contentsquare/install.yml

```
@@ -0,0 +1,19 @@
+id: third-party-contentsquare
+name: Contentsquare
+title: Contentsquare
+description: 
```

```suggestion
description: |
```
Looks like we're just missing a multi-line indicator for the YAML file!